app/js/config/environment/index.js

```diff
@@ -1,5 +1,7 @@
-if (process.env.NODE_ENV === 'production' || true) {
+if (process.env.NODE_ENV === 'production') {
```

Comment:
Can't a dotenv module handle this?.. or better still have a file-reader better still along this line
```javascript 
Object.assign(process.env, require(path.join(process.cwd(), '.env.json')))
```
where env.json is your config file